apps/socketoptions.cpp

```diff
@@ -26,55 +26,47 @@ SocketOption::Mode SrtConfigurePre(SRTSOCKET socket, string host, map<string, st
     vector<string> dummy;
     vector<string>& fails = failures ? *failures : dummy;
 
-    if ( options.count("passphrase") )
+    SocketOption::Mode mode = SocketOption::CALLER;
+    if (host != "--") // for this value it will always return CALLER
     {
-        /*
-        // Insert default
-        if ( options.count("pbkeylen") == 0 )
+        string modestr = "default";
+
+        if ( options.count("mode") )
         {
-            options["pbkeylen"] = "16"; // m_output_direction ? "16" : "0";
```

Comment:
Does this rely on the default pbkeylen value inside of SRT, instead of setting it explicitly?

Reply:
It is commented out. Missed that.